### DISCUSSION TITLE
The `speaker` property is missing from transcription response

### DISCUSSION THREAD

**Discussion by cowanAI** [deepgram/community#377] [General help] [answered]
### Which Deepgram product are you using?

None

### Details

Hello Team, we are getting an error with the [Speaker] key, last week the issue was with [Paragraph] but now we are getting errors with that particular key, also multiple outages this week, we would appreciate very much if you could assist with this matter.

Thanks

**Comment by jpvajda** [comment]
@cowanAI  Can you provide more details about the issues you are seeing in your transcription results? Providing us a sample of the output helps us to troubleshoot. 

As for your mention of outages, we did have a few degradation of service incidents this week which are reported on our [Status Page](https://status.deepgram.com/). We apologize for any inconvenience you faced with these different impacted services. the time stamps for the start and end times of those are available on the status page for further clarity.

**Reply by cowanAI** [reply]
Thanks @jpvajda thankfully after adding retries, all issues have been resolved, checked my logs and no errors yesterday which is great for our users, ocasionally, Deepgram SDK is not able to retrieve 2 separate KEYS, sometimes is [Paragraphs] or [Speaker], you can learn more on this discussion: https://github.com/orgs/deepgram/discussions/372, Luke mentioned it was due to a regression in the API. Thanks for your help and we are happy no more issues ocurred yesterday.

**Reply by lukeocodes** [reply, marked as answer]
Following your recent `paragraph` issue, we had an unrelated issue where `speaker` could also be omitted. We resolved it fast, but in deployment one of our Whisper servers did not recieve the deployment.

This caused some users on Whisper to be affected for a slightly longer than usual amount of time. But it was resolved last night, thankfully.

**Comment by cowanAI** [comment]
Can confirm, no support tickets issued today, thanks!!
